### PR TITLE
feat: change model to string param

### PR DESCRIPTION
**Pull request by cabljac**
This PR changes the model params to a simple string, to make maintenance easier with all the models being released.

There are also some docs changes.

### CODE REVIEW

File: firestore-multimodal-genai/functions/src/generative-client/generative_ai.ts [hunk 1]
```diff
@@ -41,8 +41,6 @@ export class GeminiGenerativeClient extends GenerativeClient<
 
       const base64String = await getImageBase64(options.image, 'google-ai');
 
-      console.log('base64String', base64String.length);
```
Comment (jauntybrain) [inline firestore-multimodal-genai/functions/src/generative-client/generative_ai.ts:44]: Is this log needed?

File: firestore-genai-chatbot/functions/src/config.ts [hunk 2]
```diff
@@ -102,10 +81,10 @@ function getSafetySettings(): GoogleAISafetySetting[] | VertexSafetySetting[] {
 
 const config: Config = {
   vertex: {
-    model: getModel(),
+    model: process.env.MODEL!,
```
Comment (jauntybrain) [inline firestore-genai-chatbot/functions/src/config.ts:84]: Should we be handling invalid model name in any way? I see that we don't have any try/catch blocks or logs for that and `generative-ai` doesn't seem to validate it in any way until the HTTPS [request is sent](https://github.com/google/generative-ai-js/blob/main/packages/main/src/methods/generate-content.ts#L55-L60).

**Review by jauntybrain — CHANGES_REQUESTED** [review]

File: firestore-multimodal-genai/functions/src/generative-client/generative_ai.ts (hunk 1 above)
Comment (cabljac) [reply firestore-multimodal-genai/functions/src/generative-client/generative_ai.ts:44]: nope! ill remove

File: firestore-genai-chatbot/functions/src/config.ts (hunk 2 above)
Comment (cabljac) [reply firestore-genai-chatbot/functions/src/config.ts:84]: Yeah been told they prefer not to validate it:

> I think skipping the validation would be worth having to update the extension every single time Google decides to update a model.

File: firestore-multimodal-genai/functions/src/generative-client/generative_ai.ts (hunk 1 above)
Comment (cabljac) [reply firestore-multimodal-genai/functions/src/generative-client/generative_ai.ts:44]: Oh this PR is removing the log haha

File: firestore-genai-chatbot/functions/src/config.ts (hunk 2 above)
Comment (jauntybrain) [reply firestore-genai-chatbot/functions/src/config.ts:84]: Gotcha! Also fair point 😂

File: firestore-multimodal-genai/functions/src/generative-client/generative_ai.ts (hunk 1 above)
Comment (jauntybrain) [reply firestore-multimodal-genai/functions/src/generative-client/generative_ai.ts:44]: My bad!

**Comment by jauntybrain** [comment]
LGTM

**Review by jauntybrain — APPROVED** [review]

**Pull request merged**